examples/analytics-vendors.amp.html

```diff
@@ -610,6 +610,18 @@
 </amp-analytics>
 <!-- End Mediator.media example -->
 
+<!-- Mobify tracking -->
```

Comment:
Please add your name to this [form](https://github.com/ampproject/amphtml/blob/master/examples/analytics-vendors.amp.html#L45) as well for us to find your test more convienient. Thanks.

Reply:
Done 👍